docs/core/tools/dotnet-store.md

```diff
@@ -0,0 +1,78 @@
+---
+title: dotnet-store command | .NET Core SDK
+description: The dotnet-store Stores the specified assemblies in the runtime package store.
+keywords: dotnet-store, dotnet-publish, CLI, CLI command, .NET Core
+author: bleroy
+ms.author: beleroy
+ms.date: 4/13/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 1e8e4122-8110-4b48-afce-afffb6737776
+---
+
+#dotnet-store
+
+## Name
+
+`dotnet-store` - Stores the specified assemblies in the [runtime package store](../deploying/runtime-package-store.md).
+
+## Synopsis
+
+`dotnet store [--manifest] [--framework-version] [--output] [--working-dir] [--preserve-working-dir] [--skip-optimization] [-v|--verbosity] [-h|--help]`
```

Comment:
> Should it have been the argument for the command rather than a named option then?

@ramarag can answer that.  I tried getting it that way during the code review, but he had objections.